src/Chirp.DBService/Contexts/ChirpDBContext.cs

```diff
@@ -21,6 +19,10 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
             .WithMany(a => a.Cheeps)
             .HasForeignKey("AuthorId")
             .IsRequired();
+        modelBuilder.Entity<Author>().Property(a => a.Name).HasMaxLength(50).IsRequired(); //Restriction of maximum length
+        modelBuilder.Entity<Author>().Property(e => e.Email).HasMaxLength(50).IsRequired(); //Restriction of maximum length
```

Comment:
Is it possible to research whether or not we can also add a regex restriction that the email is a valid email?

Reply:
On it!